### PR TITLE
Direct user to email PDS Operator

### LINKED ISSUE #145: Feedback widget does not direct users to pds-operator@jpl.nasa.gov
## Checked for duplicates
Yes - I've already checked

## 🐛 Describe the bug
The website currently displays a feedback widget that no longer serves its intended purpose and does not route user inquiries to the PDS operations team directly. Users submitting feedback through the widget are not being directed to the appropriate contact channel.

## 🕵️ Expected behavior
The feedback widget should be replaced with a direct mailto link (`mailto:pds-operator@jpl.nasa.gov`) so that users can contact the PDS operations team directly via email without relying on a third-party widget.

## 📜 To Reproduce
1. Navigate to the PDS portal website
2. Locate the feedback widget
3. Observe that it does not direct users to `pds-operator@jpl.nasa.gov`

## 🖥 Environment Info
- Website: PDS Portal

## 📚 Version of Software Used
N/A

## 🩺 Test Data / Additional context
Replace the feedback widget with a link to `mailto:pds-operator@jpl.nasa.gov`. This ensures users are directed to the correct support email address.

## 🦄 Related requirements
N/A

---

### PR DESCRIPTION
**Pull request by c-suh**
### 🤖 AI Assistance Disclosure
- [x] No AI assistance used
- [ ] AI used for light assistance (e.g., suggestions, refactoring, documentation help, minor edits)
- [ ] AI used for moderate content generation (AI generated some code or logic, but the developer authored or heavily revised the majority)
- [ ] AI generated substantial portions of this code

Estimated % of code influenced by AI: ___ %

## ♻️ Related Issues

Resolves https://github.com/NASA-PDS/portal/issues/145

## 🤓 Reviewer Checklist

*Reviewers: Please verify the following before approving this pull request.*

### Documentation and PR Content
- [ ] **Documentation:** README, Wiki, or inline documentation (Sphinx, Javadoc, Docstrings) have been updated to reflect these changes.
- [ ] **Issue Traceability:** The PR is linked to a valid GitHub Issue
- [ ] **PR Title:** The PR title is "user-friendly" clearly identifying what is being fixed or the new feature being added, that if you saw it in the Release Notes for a tool, you would be able to get the gist of what was done.

### Security & Quality
- [ ] **SonarCloud:** Confirmed no new High or Critical security findings.
- [ ] **Secrets Detection:** Verified that the Secrets Detection scan passed and no sensitive information (keys, tokens, PII) is exposed.
- [ ] **Code Quality:** Code follows organization style guidelines and best practices for the specific language (e.g., PEP 8, Google Java Style).

### Testing & Validation
- [ ] **Test Accuracy:** Verified that test data is accurate, representative of real-world PDS4 scenarios, and sufficient for the logic being tested.
- [ ] **Coverage:** Automated tests cover new logic and edge cases.
- [ ] **Local Verification:** (If applicable) Successfully built and ran the changes in a local or staging environment.

### Maintenance
- [ ] **Backward Compatibility:** Confirmed that these changes do not break existing downstream dependencies or API contracts (or that breaking changes are clearly documented).

### CODE REVIEW

File: feedback/js/feedback.js [hunk 1]
```diff
@@ -325,8 +82,9 @@ document.addEventListener("DOMContentLoaded", function(){
...
 
 				setText: function(el, label, fontSize) {
-					var p = document.createElement("p");
-					p.append( document.createTextNode(label) );
+					var a = document.createElement("a");
+					a.setAttribute("href", "mailto:pds-operator@jpl.nasa.gov?subject=Feedback for pds.nasa.gov");
+					a.append( document.createTextNode(label) );
 					if ( fontSize !== "16" ) {
 						if ( !isNaN(fontSize) ) {
 							el.setAttribute("class", "noImage");
```
[the remark this answers is not in the document]
Comment (c-suh) [reply feedback/js/feedback.js]: Addressed the latter point in the CSS file. Ignoring the first point about pds_ versus pds- because the README and php file are outdated.

**Review by jordanpadams — APPROVED** [review]
